CHANGELOG.md

```diff
@@ -53,7 +53,7 @@ server:
   ```
   In addition, other existing uplink env variables are now also configurable via arg. 
 
-- **Make deduplication and caching more robust against cancellation** [PR #752](https://github.com/apollographql/router/pull/752)
+- **Make deduplication and caching more robust against cancellation** [PR #752](https://github.com/apollographql/router/pull/752) [PR #758](https://github.com/apollographql/router/pull/758)
```

Comment:
Probably need a new title here. The nature of this change is to "eliminate memory leaks when tasks are cancelled" (or something like that).

Reply:
++